hw1/02-palindrome.js

```diff
@@ -1,3 +1,27 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
+const output = document.getElementById("output");
 
-elem.addEventListener('input', handleInput);
+handleInput = () => {
```

Comment:
Nice work using an anonymous function here, the AirBnB style guide suggests slightly different syntax however. Might want to look that up. I used a linter with my project and it helped with some of the more minor style syntax errors.

Reply:
Thanks for letting me know. I have updated the functions in latest commit.